### ISSUE TITLE
Cannot use tasty with GHCJS (at least clock is problematic)

### ISSUE THREAD

**Issue by phadej** [feuerbach/tasty#288]
```
uncaught exception in Haskell main thread: ReferenceError: h$clock_gettime is not defined
ReferenceError: h$clock_gettime is not defined
```

**Comment by feuerbach** [comment]
I pushed a potential fix to the `fix-ghcjs` branch, could you try it?

**Comment by phadej** [comment]
That seems to work!

**Comment by feuerbach** [comment]
Cool, released as tasty-1.4.0.2.

**Comment by phadej** [comment]
Thanks!
